CHANGELOG.md

```diff
@@ -7,7 +7,7 @@ Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## [Unreleased]
 
-- Nothing yet!
+- Brotli support added by Dr. Matthew Swabey
```

Comment:
Do you want to link to your GitHub profile here?

- Brotli support added by [Dr. Matthew Swabey](https://github.com/mattaw)

You don't have to of course, just throwing the idea out there.